packages/dependency-extraction-webpack-plugin/assets/packages.js

```diff
@@ -16,6 +16,7 @@ module.exports = [
 	'@woocommerce/tracks',
 	// wc-blocks packages
 	'@woocommerce/blocks-checkout',
+	'@woocommerce/block-data',
```

Comment:
I see https://www.npmjs.com/package/@woocommerce/block-data returns 404. If this hasn't been published before, can we follow the naming convention, ie block v. blocks?

Reply:
The name we use internally is `@woocommerce/block-data`, changing this might require will backward computability.